go.mod

```diff
@@ -20,10 +20,10 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lib/pq v1.10.9
 	github.com/lightninglabs/aperture v0.3.2-beta
-	github.com/lightninglabs/lndclient v0.18.0-1
+	github.com/lightninglabs/lndclient v0.18.4-0
 	github.com/lightninglabs/loop/looprpc v1.0.0
 	github.com/lightninglabs/loop/swapserverrpc v1.0.8
-	github.com/lightningnetwork/lnd v0.18.3-beta.rc1
+	github.com/lightningnetwork/lnd v0.18.3-beta.rc3.0.20241011124628-ca3bde901eb8
```

Comment:
Did you mean v0.18.4-beta?

Reply:
We don't have a tag for that yet. This is a commit in https://github.com/lightningnetwork/lnd/pull/9183, which should remain stable (meaning I won't rebase that branch, only add more commits from new cherry-picked commits).
So we'll need another PR to bump to the final, tagged version of `lnd 0.18.4-beta`, but that hopefully will not include any other code changes, only a `go.mod/go.sum` bump.